src/release/utils/configBuilder.ts

```diff
@@ -0,0 +1,68 @@
+import type { JSONSchemaType } from 'ajv';
+import Ajv from 'ajv';
+import type {
+  ProjectReleaseConfig,
+  ProjectReleaseConfigJSON,
+} from '../typings/ProjectReleaseConfig';
+import type { ReleaseManager } from '../typings/ReleaseManager';
+import type { ReleaseTagManager } from '../typings/ReleaseTagManager';
+
+// only one instance should be used across all project
```

Comment:
Do you mean homer? => "all the project"
Or do you mean all the projects in the config file? => "all projects"